Sources/Extensions/String+Array.swift

```diff
@@ -42,6 +42,17 @@ extension String {
 
         return ceil(boundingBox.height)
     }
+
+    func currencyFormatter(by currencyCode: String) -> String {
```

Comment:
There's another ticket to format amount. Let's remove this logic from here and handle it in another ticket

Reply:
Sure. I will remove it for now.